README.md

```diff
@@ -94,8 +84,20 @@ To update these files run `make apidocs`
 
 To render the docs to HTML run `make docs`
 
-For admins
-----------
+
+
+## Things that might be relevant for more advanced users/developers
```

Comment:
We run a check for black and run all tests when anyone opens a PR. Doesn't that make this relevant for everyone? People will skip this and when the checks fail I am afraid they will have no clue on what to do.